src/tests/login-test.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('message controllers', () => {
+  it('logs in a user ', (done) => {
+    chai.request(app)
+      .post('/login')
+      .field({
+        email: 'Mcstain1639@gmail.com',
+        password: 'Mc1639_1639',
+      })
+      .end((err, res) => {
+        res.body.should.be.a('object');
+        res.body.should.have.status(200);
```

Comment:
These are the only assertions you have in all the test cases. Can you please expand these to cover different things like what gets returned or essentially what you look out for when you're testing an endpoint in Postman?